.gitignore

```diff
@@ -39,6 +39,9 @@ benchmark/mysql-connector-cpp/
 benchmark/mariadb.json
 benchmark/mysql.json
 benchmark/main-benchmark
+cmake-build-*/
+build/
```

Comment:
These are too project-specific. I am not sure I want these to be added to .gitignore for everybody. On other name it's pretty common naming pattern. Once again - I am not sure about this